src/AWS.Logger.Core/Core/AWSLoggerCore.cs

```diff
@@ -211,7 +211,9 @@ private string GetServiceUrl()
             try
             {
                 _client.Config.Validate();
+#pragma warning disable CS0618 // Type or member is obsolete
                 return _client.Config.DetermineServiceURL() ?? "Undetermined ServiceURL";
```

Comment:
Could we use an updated API instead of disabling the warning?

Reply:
The updated API is [DetermineServiceOperationEndpoint(AmazonWebServiceRequest request)](https://github.com/aws/aws-sdk-net/blob/46a53baed18c9aecde8a173d9c8af51c108f99a4/sdk/src/Services/CloudWatchLogs/Generated/_netstandard/AmazonCloudWatchLogsClient.cs#L5021). This isn't a drop-in replacement because of the new `AmazonWebServiceRequest` argument.